Tuya config options (#15388)
home-assistant/home-assistant.github.io: source/_integrations/tuya.markdown

- **Scene** - The device state in frontend panel will not change immediately after you activate a scene.
- **Switch** - The platform supports switch and socket.

> 

Reviewer: ```suggestion
```

  default: tuya
{% endconfiguration %}

## Integration Options

> Through the integration options it is possible to change various behaviors, some common for integration and others specific to each `light` and `climate` devices. These can be changed at **Tuya** -> **Options** on the Integrations page.

Reviewer: ```suggestion
It is possible to change various behaviors through the integration options, some common for integration and others specific to each `light` and `climate` devices. These can be changed at **Tuya** -> **Options** on the Integrations page.
```

### Common Options

> - **Discovery device polling interval** (default=605): define the interval between 2 consecutive call to `API discovery method`, that is used to get the status for all 
> registered devices with single call. If you set interval value too low, Tuya API will return errors so it is suggested to use default value until
> you now that is possible to use lower values.

Reviewer: ```suggestion
- **Discovery device polling interval** (default=605): define the interval between 2 consecutive calls to the `API discovery method`, which is used to get the status for all registered devices with a single call. If you set interval value too low, Tuya API will return errors, so it is suggested to use the default value until
you know that it is possible to use lower values.
```

> - **Query device polling interval** (default=120): this option is available only if you have devices that can use `API query method`. 
> It define the interval between 2 consecutive call to `API query method`, that is used to get the status for a specific device. 
> This method is always used when is available only one device that can use it. If you set interval value too low, Tuya API will return errors 
> so it is suggested to use default value until you now that is possible to use lower values.

Reviewer: ```suggestion
- **Query device polling interval** (default=120): this option is available only if you have devices that can use the `API query method`. 
It defines the interval between 2 consecutive calls to the `API query method`, that is used to get the status for a specific device. 
This method is always used when it is available only one device that can use it. If you set interval value too low, Tuya API will return errors 
so it is suggested to use the default value until you know that is possible to use lower values.
```

> - **Device that will use query method**: this option is available only if you have devices that can use `API query method`. 
> Because it is not possible make multiple call to `API query method`, if you have more than one device that can use it you can choose which 
> one will use. This will give better status refresh for this specific device.

Reviewer: ```suggestion
- **Device that will use the query method**: this option is available only if you have devices that can use the `API query method`. 
Because it is not possible to make multiple calls to the `API query method`. If you have more than one device that can use it you can choose which one will use. This will give a better status refresh for this specific device.
```

> - **Device to configure (multi-select list)**: this option is available only if you have `light` or `climate` device. Selecting a device to 
> configure to option page related to the device will be opened. You can also select more than one devices to configure them simultaneously, 
> but all selected devices must be of the same type.

Reviewer: ```suggestion
- **Device to configure (multi-select list)**: this option is available only if you have a `light` or `climate` device. Selecting a device to 
configure to options page related to the device will be opened. You can also select more than one devices to configure them simultaneously, 
but all selected devices must be of the same type.
```

### Light Options

> - **Force color support**: when checked force `color support` for device that do not report this feature.

Reviewer: ```suggestion
- **Force color support**: when checked force `color support` for devices that do not report this feature.
```

- **Brightness range**: change the `brightness range` used for the device. Possible options are:
    - range 1-255 (default)
    - range 10-1000

- **Min color temperature**: set minimum `color temperature` expressed in `kelvin` accepted by the light.

- **Max color temperature**: set maximum `color temperature` expressed in `kelvin` accepted by the light.

- **Max color temperature reported**: set the maximum `color temperature` value reported by the light.

### Climate Options

> - **Temperature unit**: change the `temperature unit` used internally by the device.

Reviewer: ```suggestion
- **Temperature unit**: change the `temperature unit` used internally by the devices.
```
